amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/common/fm/FMSettings.java

```diff
@@ -0,0 +1,60 @@
+package org.digijava.kernel.ampapi.endpoints.common.fm;
+
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import org.digijava.kernel.ampapi.endpoints.common.EPConstants;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class FMSettings<T> {
```

Comment:
Let's use a different naming, may be `FMResult`? Since it has inside another `fmSettings` property which is confusing.

Reply:
agree